packages/select-rich/stories/index.stories.mdx

```diff
@@ -312,17 +312,11 @@ It is up to you how to render this Object in the DOM.
       );
```

Comment:
My Prettier went kinda nuts on this. Might be nice to change: `"format:prettier": "prettier \"**/*.{js,md}\" --write",` to `"format:prettier": "prettier \"**/*.{js,md,mdx}\" --write",`, we already have it on our extension layer since last week and it works pretty well

Reply:
Soon we won't have .mdx though so I guess it's not worth the effort now :P